run/pem2john.py

```diff
@@ -40,57 +40,85 @@
    70:d=1  hl=4 l= 640 prim: OCTET STRING      [HEX DUMP]:C4BC6BC5447BED58...
 """
 
+def is_binary(filename):
```

Comment:
Is this really necessary?

I will try to come up with an alternate approach.

Reply:
*handled it

Reply:
> 
> 
> Is this really necessary?
> 
> I will try to come up with an alternate approach.

Would it be OK for you if I made the changes as I suggest with an argv[] param?